openHarmony/openHarmony_drawing.js

```diff
@@ -440,11 +440,13 @@ $.oDrawing.prototype.importBitmap = function (file, convertToTvg) {
     var _bin = specialFolders.bin + "/utransform";
 
     var tempFolder = this.$.scn.tempFolder;
+    var res_x = this.$.scn.resolutionX
+    var res_y = this.$.scn.resolutionY
```

Comment:
could we instead use the resolution of the original image to ensure an import at the correct size?

```javascript
var _info = CELIO.getInformation(file.path);
var resX = _info.width;
var resY = _info.height;
```